src/canisters/governance/request.converters.ts

```diff
@@ -824,3 +824,21 @@ export const toIncreaseDissolveDelayRequest = ({
   ],
   neuron_id_or_subaccount: [],
 });
+
+export const toJoinCommunityFundRequest = (
+  neuronId: NeuronId
+): RawManageNeuron => ({
+  id: [{ id: neuronId }],
```

Comment:
I guess that all the request convertes that use ManageNeuron have a similar pattern and could reuse some helper, but I would still keep the `toJoinCommunityFundRequest` for the GovernanceCanister method.

In `ts` they use the proto buffer classes in `governance_pb.d.ts`. I think that we are moving away from that, but we could take a look at the patterns there in case we like them.